lib/zclient.c

```diff
@@ -1991,6 +1991,35 @@ int lm_label_manager_connect(struct zclient *zclient)
 	return (int)result;
 }
 
+/*
+ * Asynchronous label chunk request
+ */
```

Comment:
Couldn't we document the parameters in the comments like it was done for `lm_get_label_chunk()`?

Reply:
Will be in my next git push.